backend/python/app/graphql/queries/story_query.py

```diff
@@ -51,3 +59,10 @@ def resolve_story_translation_tests(root, info, language, level, stage, story_ti
     return services["story"].get_story_translation_tests(
         user, language, level, stage, story_title
     )
+
+
+def _select_user_id_for_available_translations_query(user_id):
```

Comment:
Putting this here because moving it to story_service means adding the user service to the story service. Also, this sticks to the principle of keeping authentication out of the story services. lmk if you can think of a better name tho LOL

Reply:
what do you think about something like `target_user_id` instead of `searching_user_id`? i'm not sure if it's more readable or not

and consequently something like `_target_user_id_for_available_translations_query` for the helper

Reply:
I think the verb "select" makes more sense than "target" for the function name, but I will swap searching_user_id for target_user_id !